Italian/main/TeleService.apk/res/values-it/strings.xml

```diff
@@ -1006,8 +1006,8 @@ Sarà usata al suo posto la rete preferita.</string>
     <string name="auto_answer_scenario_title">Condizione per risposta automatica</string>
     <string name="auto_answer_scenario_enteries_always">Sempre</string>
     <string name="auto_answer_scenario_enteries_headset_only">Solo auricolari</string>
-    <string name="auto_answer_scenario_enteries_bluetooth_only">Solo bluetooth</string>
-    <string name="auto_answer_scenario_enteries_headset_bluetooth">Solo auricolari o bluetooth</string>
+    <string name="auto_answer_scenario_enteries_bluetooth_only">Solo Bluetooth</string>
+    <string name="auto_answer_scenario_enteries_headset_bluetooth">Solo auricolari o Bluetooth</string>
```

Comment:
ahahah figurati. è solo che essendo praticamente un hobby per me questo della traduzione non è la prima volta che mi capita di sentire questo tipo di richieste e avendo già sperimentato le conseguenze che possono avere ho preferito metterti in guardia subito ;)